src/index.html

```diff
@@ -10,8 +10,175 @@
+  <body class="row">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="Surename">Surename:
+          </label>
+          <input type="text"
+            id="Surename"
+            name="Surname" multiple
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="Name">Name:
+          </label>
+          <input type="text"
+            id="Name"
+            name="Name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="old">How old are You?
+          </label>
+          <input type="number"
+            name="howold"
+            id="old"
+            value="12"
+            atocomplete="off"
+            max="100"
+          >
+        </div>
+        <div class="form-field">
+          <label for="date of birth">Full date of birth
+          </label>
+          <input type="date"
+            id="date of birth"
+            name="date"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="I accept the term of the agreement">I accept the term of the agreement
+          </label>
+          <input class="input" type="checkbox"
+            id="I accept the term of the agreement"
+            name="terms"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail
+          </label>
+          <input type="email" id="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="password">Password
+          </label>
+          <input class="input" type="password"
+            id="password"
+            name="password" required
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about me
+        </legend>
+        <div class="form-field">
+             Do you love cats?
+          <label>
+            <input type="radio" name="cats"
+              autocomplete="off"
+            >
+                Yes
+          </label>
+          <label>
+            <input type="radio" name="cats"
+              autocomplete="off"
+            >
+                 No
+          </label>
+
+        </div>
+        <div class="form-field">
+          <label for="color">What is your favorite color?
+          </label>
+          <input type="color"
+            id="color"
+            value="ffff"
```

Comment:
Please use a valid hex color code for the 'value' attribute, for example: '#ffffff'.